ios/Classes/FacebookLikeButton.m

```diff
@@ -20,11 +21,19 @@ -(FBSDKLikeControl*)like
 {
     if (like == nil) {
         like = [[FBSDKLikeControl alloc] init];
+        
```

Comment:
You're assigning like on line 24 then you break a line and use it. I think it looks better if you just have it a 3 line code block. But up to you, you don't have to change it.